base/applications/cleanmgr/main.c

```diff
@@ -36,54 +36,64 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance,
         return FALSE;
     }
 
-    if (Obj)
+    /* Creating a mutex of this specific program to check if multiple instances are running */
+    CreateMutexW(NULL, FALSE, L"cleanmgr.exe");
```

Comment:
1: Why is there only one instance allowed?
2: Did you test this at all?

Reply:
@learn-more
1. The cleanmgr on Windows only allows one instance to run.
2. That was my big oversight.